templates/repo/actions/runs_list.tmpl

```diff
@@ -6,31 +6,35 @@
 	</div>
 	{{end}}
 	{{range .Runs}}
-		<li class="item gt-df gt-py-4">
+		<li class="item gt-df gt-py-3">
 			<div class="issue-item-left issue-item-icon gt-df gt-items-start">
 				{{template "repo/actions/status" (dict "status" .Status.String "locale" $.locale)}}
 			</div>
-			<div class="issue-item-main action-item-main gt-f1 gt-fc gt-df gt-mr-3">
+			<div class="issue-item-main action-item-main gt-f1 gt-fc gt-df gt-mr-3 gt-gap-2">
 				<div class="issue-item-top-row gt-mb-2">
 					<a class="index gt-no-underline title action-item-title" title="{{.Title}}" href="{{if .Link}}{{.Link}}{{else}}{{$.Link}}/{{.Index}}{{end}}">
 						{{- .Title -}}
 					</a>
 				</div>
 				<div class="desc issue-item-bottom-row gt-df gt-ac gt-fw">
-					<b>{{if not $.CurWorkflow}}{{.WorkflowID}} {{end}}#{{.Index}}</b>: {{$.locale.Tr "actions.runs.commit"}}
-					<a href="{{$.RepoLink}}/commit/{{.CommitSHA}}">{{ShortSha .CommitSHA}}</a>{{$.locale.Tr "actions.runs.pushed_by"}} {{.TriggerUser.GetDisplayName}}
-					<span class="ui label">
-						{{if .RefLink}}
-							<a href="{{.RefLink}}">{{.PrettyRef}}</a>
-						{{else}}
-							{{.PrettyRef}}
-						{{end}}
+					<b>{{if not $.CurWorkflow}}{{.WorkflowID}} {{end}}#{{.Index}}</b>
+					: {{$.locale.Tr "actions.runs.commit"}}
```

Comment:
Does the container need "gap"? Otherwise the spaces in a flex container are lost?

Reply:
It did seem to work without. Generally I try to avoid `gap` for cases where the user may copy text, like this one.

Reply:
It's converted to a single text node, so it does not need `gap`:

<img width="151" alt="Screenshot 2023-07-03 at 23 56 06" src="https://github.com/go-gitea/gitea/assets/115237/1d3bed1d-0477-4c4b-ba3c-ff120a7cb09f">